index.html

```diff
@@ -16,8 +16,14 @@
 
       <header>
```

Comment:
[OPTIONAL] Cards/works area responding very well to the different screen sizes. But the heading and contact section has some bugs and looks like the below. I think it would better if you can fix it to make your web page more responsive. 📱
<img width="1010" alt="Screenshot 2023-01-20 at 8 28 57 AM" src="https://user-images.githubusercontent.com/25887253/213624648-a7679a45-5dba-4aff-954c-9090cc001481.png">